src/apis/resource/resource_api.py

```diff
@@ -14,6 +14,8 @@
     json_iri_lit_from_rdf_graph,
     json_iri_bnode_from_rdf_graph,
 )
+from apis.resource.DAANStorageLODHandler import DAANStorageLODHandler  # noqa: F401
+from apis.resource.SDOStorageLODHandler import SDOStorageLODHandler  # noqa: F401
```

Comment:
What are the implicit dependencies?

Note that, given the fact that we want the lod-server to use profile negotion, we need to determine what class the current profile (given by the client) needs. 
Before, the type definitions were done in the settings.py, which doesn't hold when using YAML. Therefore, importing the handler classes in this module seems a solution (although not perfect, because adding more profiles would imply changing the code as well. The config is not very separated from the code then). 
Also, we only read from the current scope that these globals() gives us (in line 160), so we don't change anything. 
But, when there are better solutions, I am happy to hear.

Reply:
Maybe a solution is using global variable dict defined in the module:
```python
profiles = {
    "DAANStorageLODHandler": DAANStorageLODHandler,
    "SDOStorageLODHandler": SDOStorageLODHandler,
}
```
dan verderop in de code:
```python
        resp, status_code, headers = profiles[profile["storage_handler"]](
            app_config, profile
        ).get_storage_record(level, identifier, mt.to_ld_format())

```

Reply:
Forget above. @jblom had a more dynamic solution using importlib's import_module. I will refactor using that method in  the base_util.

Reply:
Refactored globals to import_module solution in commit 114d24518aa5752eac8366a78c1603be2974ae34